sl1m:
    numerical_solver: "GUROBI"
    package: "talos-rbprm"
    limbs:
        - "LF"
        - "RF"
    suffix_com: "_effector_frame_REDUCED.obj"
    suffix_feet: "_quasi_flat_REDUCED.obj"
    gait:
        - [1, 0]
        - [0, 1]
    plot: false
    nb_steps: 30
    costs: {
        "step_size": [0.1, [0.0, 0.0]]
    }
    optimize_com: true
    rate: 10
    com_height: 0.87
    step_length: [0.4, 0.2]